### ISSUE TITLE
stocks, autodump causes "ghost items"

### ISSUE THREAD

**Issue by ab9rf** [DFHack/dfhack#1052]
From [bay12 forums](http://www.bay12forums.com/smf/index.php?topic=139553.msg7302494#msg7302494)
> I'm using x64 version, and for some reason the enhanced stockpile management menu messes up items if I mark listed items as dump. The game creates ghost items on the screen, whose positions are random and can't be checked using k. And the game tends to crash after that. Autodump seems to have similar issue.

May be related to the uninitialized bitfield constructor issue, but not sure about this.

**Comment by lethosor** [comment]
I can't seem to reproduce this in 506d124a986, even when I disable the bitfield initialization code in StructFields.pm.

**Comment by lethosor** [comment]
On second glance, this part looks pretty relevant:
```cpp
        else if (input->count(interface_key::CUSTOM_SHIFT_D))
        {
            df::item_flags flags;
            flags.bits.dump = true;
            toggleFlag(flags);
            populateItems();
        }
```

**Comment by lethosor** [comment]
I'd like to assume that this is fixed along with #1047, but we should probably make a new build and have Libash_Thunderhead check.

**Comment by lethosor** [comment]
Since I haven't seen any additional reports of this, and the issue wasn't present in alpha2, I'm going to call this fixed.
